src/encord_active/lib/db/schema.prisma

```diff
@@ -49,6 +51,6 @@ model LabelRow {
   data_type      String
   created_at     String
   last_edited_at String
-  location       String
   data_units     DataUnit[]
+  label_row_json String     @default("missing")
```

Comment:
I think making the field nullable is better than putting a string that isn't a valid json in a field that holds a json string.

```suggestion
  label_row_json String?
```